lib/publiccloud/azure.pm

```diff
@@ -100,6 +105,8 @@ sub create_resources {
     record_info('INFO', 'Create storage container ' . $self->container);
     assert_script_run('az storage container create --account-name ' . $storage_account
           . ' --name ' . $self->container, $timeout);
+    # Image gallery for Arm64 images
+    assert_script_run('az sig create --resource-group ' . $self->resource_group . ' --gallery-name "' . $self->image_gallery . '" --description "openQA upload Gallery"', timeout => 300);
```

Comment:
is it necessary to do this also for "x86_64" calls ? ( let's do this conditionally to arch )

Reply:
This is the routine for the creation of the persistent objects. This is architecture independent.

Reply:
But the `$self->resource_group` and `$self->image_gallery` are not often changing. Is it OK if this will be called multiple times with the same parameters?

Reply:
I would say that currently this function is ALWAYS skipped . We calling it only when 'openqa_upload' resource group is not exists https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/publiccloud/azure.pm#L123 . 
But I  see that it is called https://duck-norris.qe.suse.de/tests/11625#step/upload_image/71 0_o . @grisu48  what am I missing here ?

Reply:
resolved